src/terminal.c

```diff
@@ -2307,35 +2307,6 @@ terminal_loop(int blocking)
     return ret;
 }
 
-/*
- * Called when a job has finished.
- * This updates the title and status, but does not close the vterm, because
- * there might still be pending output in the channel.
- */
-    void
-term_job_ended(job_T *job)
```

Comment:
I removed this unused function. But I wonder, perhaps it was meant to be called and there is a bug (which would be a bug prior to my change).